lte/gateway/python/integ_tests/Makefile

```diff
@@ -88,8 +88,3 @@ ifdef TESTS
 else
 	$(foreach test,$(NON_SANITY_TESTS),$(call execute_test,$(test));)
 endif
```

Comment:
Hi @LKreutzer, we usually use "make status" in magma VM to check the status of all services. So it would be good idea to keep it. This change occurred here, https://github.com/magma/magma/pull/13782/files#diff-41d3ad27a8ccc4fa615212321c1f0d4d49af2a4920598f9c1283ea1856c8820bL98
So can we add here?

Reply:
Hi @rsarwad, let us continue this discussion on https://magmacore.slack.com/archives/C018J8UMGMR/p1662377712669059 please, since this topic is unrelated to this PR.